README.md

```diff
@@ -10,68 +10,77 @@ In our hypothetical scenario, we have a client who needs to track their sales, s
 
 Provided in this repository is a [CSV file containing the relevant data for your application](/data.csv).
 
+**Note:**
```

Comment:
nit - I recommend moving this below the next paragraph

Reply:
@sdpeyton changed